src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Layer5 and HashiCorp Launch Service Mesh Partnership"
+date:   2020-05-28 00:00:00 +0800
+author: Lee Calcote
+thumbnail: layer5-hashicorp.png  
+eurl: /company/news/layer5-hashicorp-launch-service-mesh-partnership
+
+---
+
+import { NewsWrapper } from "../News.style.js";git
+
```

Comment:
Import the images as `import Image1 from "./layer5-image-hub-on-hashicorp-consul.png";` like that and use it in the `img` tag instead of directly referring it.
use this https://github.com/layer5io/layer5/blob/fb711bc6abf160f7c566854a7053b8c666000090/src/collections/projects/distributed-performance-management-of-service-meshes/index.mdx
as a reference